packages/grafana-data/src/types/logs.ts

```diff
@@ -56,7 +56,6 @@ export interface LogRowModel {
   logLevel: LogLevel;
   raw: string;
   searchWords?: string[];
-  timestamp: string; // ISO with nanosec precision
```

Comment:
I removed this as it was either unclear what this will be and as right now it is expect that we get iso string with ms precision from datasource having it here does not seem important as we mostly use  `timeEpochMs ` anyway